build/Dockerfile.base

```diff
@@ -1,21 +1,16 @@
-FROM docker.io/nvidia/cuda:11.8.0-base-ubi8
+FROM registry.access.redhat.com/ubi8/ubi:8.4
```

Comment:
we still need cuda base image for x86

Reply:
is there any approach that we can install cuda by yum?

Reply:
I don't want our base image is from a specific GPU device image, otherwise when we add other device there may conflicts between devices as for container image it can't have multiple parent image.
Hence I would like to use a common image for example centos or fedora. And install device divers from the common OS.

Reply:
cuda image doesn't support s390

<img width="625" alt="image" src="https://user-images.githubusercontent.com/7062400/225924240-4d21bfd5-6b42-44c8-94c2-16e0426912a5.png">

let's make `FROM` a configurable option in the dockerfile. Default to cuda image if it is not set

Reply:
> cuda image doesn't support s390
> 
> <img alt="image" width="625" src="https://user-images.githubusercontent.com/7062400/225924240-4d21bfd5-6b42-44c8-94c2-16e0426912a5.png">
> 
> let's make `FROM` a configurable option in the dockerfile. Default to cuda image if it is not set

@rootfs , if a specific device means we need a specific base image. In my point of view that's no good. 
For other GPU usage or other specific device, our image may conflicts.
Here my suggestion is that we base on a common base image as fedora or centos.
And if needed we install the device by specific docker tag
as 
kepler:x86
kepler:x86_cuda
we need an expandable way for our docker images. Otherwise, the docker image approach will becomes platform and device specific... which meaningless.

Reply:
<img width="1593" alt="image" src="https://user-images.githubusercontent.com/7820992/225929188-a23a871d-86b8-4ac0-b693-0268ae34db29.png">
@rootfs, could you please test cuda https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&Distribution=RHEL&target_version=8&target_type=rpm_network rpm works on x86?